lib/LaTeXML/Package/beamer_support_preload.sty.ltxml

```diff
@@ -0,0 +1,482 @@
+# /=====================================================================\ #
+# |  beamer_support_preload.sty                                         | #
+# | Implementation for LaTeXML                                          | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+# This package is virtual and contains support code for beamer.
+# It's loaded by  beamer{article,}.cls *before* options are read.
+
+sub beamerTODO {
+  my ($feature) = @_;
+  Warn('todo', $feature, "beamer feature $feature is not yet supported; binding requires more development"); }
+
+#**********************************************************************
+# Custom Beamer Arguments
+#**********************************************************************
+
+# The 'Angled' (and corresponding 'OptionalAngled') parameter types are arguments in angle brackets.
+# The 'Angled' argument type is a literal argument type, meaning is not digested by default.
+# Example: <value>
+DefParameterType('Angled',         \&readAngled, reversion => \&revertAngled);
```

Comment:
For shortish `DefParameterType`, unless code is reused, I prefer to just use anonymous subs.